phidget-sys/build.rs

```diff
@@ -25,12 +25,14 @@ fn config_macos() {
 fn main() {
     // TODO: We should eventually find or regenerate the
     //      bindings file for the specific target.
-    let tgt = env::var("TARGET").unwrap();
-    println!("debug: Building for target: '{}'", tgt);
+    // PHIDGET_ROOT should be set to point to the installation directory of phidgets (e.g. C:\Program Files\Phidgets\Phidget22)
+    let phidget_libs = env::var("PHIDGET_ROOT").unwrap();
+    println!("debug: Building for target: '{}'", phidget_libs);
```

Comment:
This might be convenient or necessary on Windows, but painful on Linux where the libraries are installed to default locations. Make it so that this is optional, i.e:
```
if let Ok(phidget_root) = env::var("PHIDGET_ROOT") { ... }
```

Reply:
Added the if let condition